tests/test_validation.py

```diff
@@ -49,15 +49,16 @@ def __init__(self, *,
 def test_custom_field():
     a = A(target="http://example.com/")
 
-    data, errors = a.dump()
+    data = a.dump()
     assert data == {"target": "http://example.com/"}
 
 
 def test_custom_field_fails_validation():
-    a = A(target="example.com/")
-
-    data, errors = a.dump(strict=False)
-    assert errors == {"target": ["Not a valid URL."]}
+    try:
+        a = A(target="example.com/")
```

Comment:
Test won't fail if there is no validation failure.

Reply:
I'm not sure what you mean here.

Reply:
If no exception is thrown (which I believe is the case), there is no assert statement being executed.